docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_6.rst

```diff
@@ -44,28 +40,30 @@ Release notes: Open GEE 5.3.6
       .. rubric:: Resolved Issues
 
       .. list-table::
-         :widths: 10 30 55
+         :widths: 35 65
          :header-rows: 1
 
          * - Number
            - Description
-           - Resolution
-         * - 9999
-           - Add description
-           - Add resolution
+         * - 1851
+           - Open GEE does not function properly on systems with a restrictive umask
+         * - 1850
+           - Fusion should include an option to set secure permissions for special folders

```

Comment:
This is the same as the change listed under "New Features", right? I don't think it needs to be both places.